packages/cli/src/util/telemetry/send-telemetry.ts

```diff
@@ -0,0 +1,20 @@
+/**
```

Comment:
Is this file still necessary? I thought we decided on implementing as a CLI subcommand?